### ISSUE TITLE
Can't run spgw without gtp?

### ISSUE THREAD

**Issue by dougbtv** [dougbtv/oai-ansible#2]
Instructions are unclear on this one, we install the rt kernel for openair-cn, but, it says you run three processes, including spgw -- which complains that:

```
[root@oai1 scripts]# ./run_spgw 
[... snip ...]
rmmod: ERROR: Module gtp is not currently loaded
modprobe: FATAL: Module gtp not found.
000143 00000:876648 7FAFFD1B6740 CRITI GTPv2- air-cn/src/gtpv1-u/gtpv1u_task.c:0105    ERROR in loading gtp kernel module (check if built in kernel)
000144 00000:876686 7FAFFD1B6740 ALERT SPGW-A rc/openair-cn/src/sgw/sgw_task.c:0151    Initializing GTPv1-U ERROR
```

Unsure what's necessary, and omitting this from running for now.

**Comment by dougbtv** [comment]
Running in the `enb.yaml` playbook

**Comment by zenghui-shi** [comment]
my understanding of this question is that we don't need real time on openair-cn node which shall be installed with >4.11 kernel from ElRepo and has spgw, hss, mme running on it. for the oaisim node which runs eNB + static UE , it requires real time kernel and configuration.

comments from Cedric Roux's email:

> This explains how to run the core network (openair-cn)
> in one machine and oaisim in another machine.
> 
> oaisim is a full stack implementation of UEs and eNBs,
> including channel modelling and traffic generation/scenario
> management. In my case, I run a basic configuration:
> 1 eNB + 1 static UE.

comments from Frank:
> the RHEL/CentOS rt-kernel is only needed for the eNB software (openairinterface5g).
> The EPC software (openair-cn) does not need rt.

based on above:
openair-cn node runs EPC softwares which are hss + mme + spgw
oaisim node is equal to openairinterface5g node which runs eNB + static UE

**Comment by dougbtv** [comment]
Fixed in #4
